ethtool: Add support of coalesce

cathay4t opened the pull request:
Example:

```yml
--- interfaces:
- name: sim0
  state: up
  ethtool:
    coalesce:
      rx-usecs: 100
```

Unlike ethtool feature, nmstate is using the name chose by ethtool CLI
because kernel does not have string for them.

Integration test case included. But only on Fedora 34 where netdevsim
support coalesce.

Unit test cases included.

ffmancera reviewed (dismissed):
Thanks for all the great work of ethtool!

cathay4t committed:
ethtool: Add support of coalesce

Example:

```yml
---
interfaces:
  - name: sim0
    state: up
    ethtool:
      coalesce:
        rx-usecs: 100
```

Unlike ethtool feature, nmstate is using the name chose by ethtool CLI
because kernel does not have string for them.

Integration test case included. But only on Fedora 34 where netdevsim
support coalesce.

Unit test cases included.

Signed-off-by: Gris Ge <fge@redhat.com>

ffmancera reviewed (approved):
Looks good, thank you!
